special-pages/pages/new-tab/app/privacy-stats/components/PrivacyStats.js

```diff
@@ -84,30 +84,25 @@ function PrivacyStatsConfigured({ parentRef, expansion, data, toggle }) {
  * @param {object} props
  * @param {Expansion} props.expansion
  * @param {TrackerCompany[]} props.trackerCompanies
- * @param {number} props.totalCount
  * @param {() => void} props.onToggle
  * @param {import("preact").ComponentProps<'button'>} [props.buttonAttrs]
  */
-export function Heading({ expansion, trackerCompanies, totalCount, onToggle, buttonAttrs = {} }) {
-    const { t } = useTypedTranslation();
+export function Heading({ expansion, trackerCompanies, onToggle, buttonAttrs = {} }) {
+    const { t } = useTypedTranslationWith(/** @type {enStrings} */ ({}));
```

Comment:
I'm not 100% happy with this approach (yet) - but it works for now. The idea is that we want some type-safety on strings, but we don't want to include duplicates of the JSON in the bundles. This is a time-sensitive workaround that won't be difficult to reverse/change if we need to.